src/components/controls/panel-layout.js

```diff
@@ -27,27 +27,31 @@ class PanelLayouts extends React.Component {
 
     return (
       <div style={{marginTop: 0, marginBottom: 10}}>
-        <PanelsFullIcon width={22} selected={this.props.panelLayout === "full"}/>
-        <SidebarButton
-          selected={this.props.panelLayout === "full"}
+        <span
+          style={{cursor: "pointer"}}
           onClick={() => {
             analyticsControlsEvent("change-layout-full");
             this.props.dispatch({ type: CHANGE_PANEL_LAYOUT, data: "full" });
           }}
         >
-          <ButtonText>{t("sidebar:full")}</ButtonText>
-        </SidebarButton>
+          <PanelsFullIcon width={22} selected={this.props.panelLayout === "full"}/>
+          <SidebarButton selected={this.props.panelLayout === "full"}>
+            <ButtonText>{t("sidebar:full")}</ButtonText>
+          </SidebarButton>
```

Comment:
(nit)

A button nested in a clickable span is presented as two separate selectable items to a screen reader (or [Vimium](https://vimium.github.io) in my case). This is what I see on the PR review app:

![image](https://github.com/user-attachments/assets/b58aac2e-70b5-427d-af9c-4cbde41a0e09)

The same desired effect can be achieved by placing the icon within the button while maintaining a single selectable item and adjusting the styles:

![image](https://github.com/user-attachments/assets/834f93ff-c1af-4ff3-a5b5-4dc8dc6f4ea5)

```jsx
<SidebarButton
  selected={this.props.panelLayout === "full"}
  style={{cursor: "pointer"}}
  onClick={…}
>
  <PanelsFullIcon width={22} selected={this.props.panelLayout === "full"}/>
  <ButtonText>{t("sidebar:full")}</ButtonText>
</SidebarButton>
```

Reply:
> By placing the icon within the button instead

Yeah, I tried that, the styling got janky. I will look at it again tomorrow.

Reply:
FSVO "tomorrow" — styling still isn't _quite_ identical, but I'm not sure the difference that's there is worth sweating over further? 

Opinions (and alternative implemantions) welcome...

Reply:
Here's my attempt at styling. Less diff compared to original code: ef5387f8bf63baded997c613ef43c559b452e496

<img width="259" alt="image" src="https://github.com/user-attachments/assets/9f24111a-a64e-47b8-a865-9a58bfe92ac7">


Not easy to get rid of the added whitespace which comes from `SidebarButton`'s [margin](https://github.com/nextstrain/auspice/blob/ef5387f8bf63baded997c613ef43c559b452e496/src/components/controls/styles.js#L74) since it's used elsewhere too. But I think it's good enough.

Reply:
squashed that commit onto this branch and force-pushed.